### PR TITLE
[No JIRA] Clarify US English use in code

### PR DESCRIPTION
**Pull request by k0nserv**
Remember to include the following changes:
+ [x] `UNRELEASED.yaml`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: decisions/writing-docs.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 Use these guidelines when writing docs for components:
 
 ### General
-* Use British English for docs, US English for code.
+
+* Use British English for docs, US English for APIs and code interfaces. Doc comments should use British English except when referring to API interfaces which should be quoted verbatim.
```
Comment (olliecurtis) [inline decisions/writing-docs.md]:
```suggestion
* Use British English for docs, US English for APIs and code interfaces. Documentation comments should use British English except when referring to API interfaces which should be quoted verbatim.
```

Comment (georgegillams) [reply decisions/writing-docs.md]: +1

Comment (k0nserv) [reply decisions/writing-docs.md]: Allright then

Comment (olliecurtis) [reply decisions/writing-docs.md]: 👍

**Comment by olliecurtis** [comment]
Forgot my approval gif

**Pull request merged**